test/configuration_test.rb

```diff
@@ -137,6 +141,26 @@
       it 'logs that the first store failed' do
         @log.string.must_match /RemoteFiles::Error/
       end
+
+      it 'adds the errors into the file'  do
+        io = StringIO.new
+        io.puts @file.errors[0]
+        io.string.must_match /RemoteFiles::Error/
```

Comment:
What type is `@file.errors[0]`? Can we not just do `@file.errors[0].to_s.must_match /RemoteFiles::Error/`? (or even without the `to_s`) Or can we use `assert_match` instead?

Reply:
Completely blanked on using `.to_s` I was seeing how the test for logs was testing the match. I can change it all to `.to_s`